Source/WebCore/rendering/RenderBlockFlow.cpp

```diff
@@ -3799,7 +3799,7 @@ static void setFullRepaintOnParentInlineBoxLayerIfNeeded(const RenderText& rende
     }
     if (!parent->isInline() || !parent->hasLayer())
         return;
-    downcast<RenderLayerModelObject>(*parent).checkedLayer()->setRepaintStatus(RepaintStatus::NeedsFullRepaint);
+    downcast<RenderLayerModelObject>(*parent).checkedLayer()->setRepaintStatus(NeedsFullRepaint);
```

Comment:
Can we do the opposite fix, leaving this as an `enum class` and adding some type casts?